src/Amadevus.RecordGenerator.Attributes/DefaultRecordFeaturesAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Amadevus.RecordGenerator
+{
+    /// <summary>
+    /// This attribute defines default <see cref="Features"/> across assembly
+    /// for usages of the non-parametrized <see cref="RecordAttribute()"/>
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = false)]
+    [Conditional("CodeGeneration")]
+    public sealed class DefaultRecordFeaturesAttribute : Attribute
```

Comment:
You can [get away with `Features` constants in the user project](https://github.com/amis92/RecordGenerator/issues/45#issuecomment-514606896) so I would postpone adding this in a future version after some field usage. I would recommend, though, adding a note to the documentation with an example along the lines of:

```c#
static class RecordFeatures
{
    public const Features Basic = Features.Constructor
                                | Features.Withers
                                | Features.Equality;

    public const Features Internal = Basic;
    public const Features Public   = Basic | Features.Builder;
}

[Record(RecordFeatures.Public)]
partial class Foo {}

[Record(RecordFeatures.Internal)]
partial class Bar {}
```

Reply:
At first I wanted to defend my idea (obviously), but then I realized that the assembly-default fallback is kind-of magical, and even though I documented it, that still doesn't feel great.

There's one scenario when magic will happen anyway because if referenced Attributes package is older than Generators used, unspecified-features default will actually use the Default known to Generators and not the one in Attributes - but since it'll only happen when versions will differ, I'm not that concerned.

Reply:
The documentation will be updated as part of #72

Reply:
Also, assembly-default got dropped.

Reply:
> There's one scenario when magic will happen anyway because if referenced Attributes package is older than Generators used, unspecified-features default will actually use the Default known to Generators and not the one in Attributes

This is a really good point! I didn't think of that and I can't help but feel that there may be more non-obvious implications here. 🤔